### PR TITLE
Updated to use new issue forms

### PR DESCRIPTION
**Pull request by entity**
Hi team,

I've updated the issue template to utilise GitHubs new issue forms, ensures certain fields are entered and allows for lots more expansion in the future - I've also attached some images of what it looks like for reference.

Thanks to the entire warp team, warp is such a fantastic product and I've loved using it already! 🎉 
CleanShot 2021-08-20 at 8 54 08@2x
CleanShot 2021-08-20 at 8 54 17@2x

### CODE REVIEW

**Comment by charlespierce** [comment]
This looks awesome for submitting bug reports! Is there a way to have something similar for new feature requests?

**Comment by heychazza** [comment]
@charlespierce oops, I completely forgot we had feature requests here, haha. Changed those too!

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    validations:
+      required: false
+  - type: dropdown
+    id: "os"
+    attributes:
+      label: "Operating System"
+      multiple: false
+      options:
+        - MacOS
+        - Windows
```
Comment (zachlloyd) [inline .github/ISSUE_TEMPLATE/bug_report.yml:49]: nit - we only support mac for now, so we can remove this

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: "Create a report to help us improve."
```
Comment (zachlloyd) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]: maybe add something along the lines of "we'd love if you'd first take a look through our open issues to make sure the issue isn't already filed"

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name: Feature Request
+description: "Suggest an idea for this project."
```
Comment (zachlloyd) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: ditto - maybe ask them to first take a look at existing feature requests and upvote if it's already filed.

**Comment by zachlloyd** [comment]
This is awesome, thank you!

LGTM

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Milo123459) [reply .github/ISSUE_TEMPLATE/bug_report.yml:49]: dont remove it in my opinion as it'll be useful for when cross platform support lands

Comment (heychazza) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Resolved as per latest commit.

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 3 above)
Comment (heychazza) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: Resolved as per latest commit.

**Comment by heychazza** [comment]
Hi @zachlloyd, I've gone ahead and updated the descriptions, and removed the other desktop OS from the dropdown. Let me know how these are, or if there are any additional changes to be made to the descriptions.

**Pull request merged**